README.md

```diff
@@ -66,11 +66,17 @@ ols_config:
   default_provider: dummy
   default_model: dummymodel
   query_validation_method: llm
+  user_data_collection:
+    feedback_disabled: false
+    feedback_storage: "/app-root/tmp/data/feedback"
```

Comment:
quick question, if i run this rcs config locally; is this path relative since other paths in the file are relative AFAIK?